Formula/xapian.rb

```diff
@@ -39,6 +28,9 @@ class Xapian < Formula
   deprecated_option "ruby" => "with-ruby"
 
   depends_on :python => :optional
+  depends_on "sphinx-doc" if build.with? "python" => :build
+  depends_on "php56" if build.with? "php"
```

Comment:
This would have to be `homebrew/php/php56` but I think we'd prefer to leave it built against the system.